client/state/initial-state.js

```diff
@@ -16,6 +17,8 @@ import config from 'config';
  */
 const debug = debugModule( 'calypso:state' );
 
+export const MAX_AGE = ms( '7 days' );
```

Comment:
>Perhaps something like this is clear enough?

Not very clear to me :smile: 

I'd suggest:

```jsx
const MAX_AGE = 604800000; // 7 days
```

Or if you think it's important to explain the conversion:

```jsx
const MAX_AGE = 604800000; // 7 days = 7 * 24 * 60 * 60 * 1000
```